x-pack/plugins/security_solution/public/management/components/endpoint_action_failure_message/index.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export { EndpointActionFailureMessage } from './endpoint_action_failure_message';
+export { EndpointActionFailureMessage, hasKnownErrorCode } from './endpoint_action_failure_message';
```

Comment:
hmmmm. this will likely cause a ci error - `hasKnownErrorCode()` was reverted

Reply:
cleaned up 5f628cd416e70997a916b172c8b52fedd6c0b062